word_search.py

```diff
@@ -0,0 +1,146 @@
+class Dictionary:
+    """
+    Input: word: string
+    Returns whether the given string is a valid word.
+    """
+    def isWord(self, word):
+        pass
+    """
```

Comment:
Please put the docstring just after the definition of the method, e.g.:

def isPrefix(self, prefix):
"""
<docstring>
"""
...